backend/tracim_backend/lib/core/content.py

```diff
@@ -1045,7 +1046,9 @@ def get_jpg_preview_path(
                 height=height,
                 file_ext=file_extension,
             )
-        except UnsupportedMimeType as exc:
+        except (PreviewDimNotAllowed, PageOfPreviewNotFound) as exc:
+            raise exc
```

Comment:
If this except is only for reader understanding, can you add a comment to explain why these exceptions can be throwed ?

Reply:
done